packages/visibility_detector/test/widget_test.dart

```diff
@@ -8,7 +8,6 @@ import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:visibility_detector/visibility_detector.dart';
-
```

Comment:
I'd prefer leaving this blank line.

Reply:
I must have accidentally removed it. I have re-added it now, too.